### PR TITLE
[Docs] Init a cookbook

### PR DESCRIPTION
**Pull request by loic425**

### CODE REVIEW

File: docs/cookbook/admin_panel/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        
+    ]);
+};
+
+```
+
+```twig
+{# templates/shared/crud/common/sidebar/logo/image.html.twig #}
+
+<img src="{{ asset('images/logo.png') }}" alt="CFC" class="navbar-brand-image" />
```
Comment (oallain) [inline docs/cookbook/admin_panel/index.md:47]:
```suggestion
<img src="{{ asset('images/logo.png') }}" alt="Your Brand name" class="navbar-brand-image" />
```

IMO, it's nore suitable ;)

File: docs/cookbook/admin_panel/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# How to customize your admin panel
+
+## How to customize the sidebar logo
+
+To customize the sidebar logo, you need to configure the new template to use using the following configuration.
```
Comment (diimpp) [inline docs/cookbook/admin_panel/index.md:5]: `to use using` -- English not englishing. :slightly_smiling_face: 
 
for example,
```suggestion
To customize the sidebar logo, you need to set new logo template at `sylius_admin.common.component.sidebar.logo` twig hook.
```

File: docs/cookbook/admin_panel/index.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+```twig
+{# templates/shared/crud/common/sidebar/logo/image.html.twig #}
+
+<img src="{{ asset('images/logo.png') }}" alt="Your Brand name" class="navbar-brand-image" />
+  
+```
+
+## How to customize the login page logo
+
+To customize the login page logo, you need to configure the new template to use using the following configuration.
```
Comment (diimpp) [inline docs/cookbook/admin_panel/index.md:53]: Same

File: docs/cookbook/admin_panel/index.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+```twig
+{# templates/shared/crud/common/sidebar/logo/image.html.twig #}
+
+<img src="{{ asset('images/logo.png') }}" alt="Your Brand name" class="navbar-brand-image" />
+  
+```
+
+## How to customize the login page logo
+
+To customize the login page logo,you need to set new logo template at `sylius_admin.security.login.logo` twig hook.
```
Comment (diimpp) [inline docs/cookbook/admin_panel/index.md:53]:
```suggestion
To customize the login page logo, you need to set new logo template at `sylius_admin.security.login.logo` twig hook.
```

**Review by diimpp — APPROVED** [review]

**Pull request merged**